backend/tests/integration/matcher/test_tasks.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from helpers.utils import create_record
+from mock import patch
+
+from inspirehep.matcher.tasks import match_references_by_uuids
+from inspirehep.records.api import LiteratureRecord
+
+
+@patch("inspirehep.matcher.tasks.match_references")
```

Comment:
why mock this? can't you have a real test that does the matching?

Reply:
wouldn't that be hard to setup? and depend on internal implementation of `reference_matcher` which is already tests and not the concern of this task.

Reply:
why hard to set up? all you need is the cited record to be available, give it some easily cited info (e.g. a DOI), then cite it by the DOI. And isn't the point of an integration test to check that everything is well integrated? I would avoid mocking stuff if it's not strictly needed.

Reply:
I would prefer to have an e2e that tests from cli instead of doing this actually.

Well the way we use integration tests sometimes, is as unit tests with DB, ES etc.